config/builds.js

```diff
@@ -37,7 +37,9 @@ const builds = buildConfigs
   })
   .map(buildConfig => {
     const name = buildConfig.name || '';
-    const env = buildConfig.env || {};
+    const env = buildConfig.env || {
+      SKU_TENANT: args.tenant
```

Comment:
Might be worth defaulting to an empty string, like we do with the `name` config.